Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Collections.Specialized;
 using System.ComponentModel;
+using System.Linq;
```

Comment:
It's necessary for the `Contains` call on `IndexPathsForSelectedRows`. However, it could be converted into a foreach.